aten/src/ATen/Context.cpp

```diff
@@ -149,6 +149,14 @@ bool Context::isXNNPACKAvailable() const {
 #endif
 }
 
+bool Context::releaseOriginalWeights() const {
+  return release_original_weights;
+}
+
+void Context::setReleaseOriginalWeights(bool e) {
+  release_original_weights = e;
```

Comment:
Is this thread safe?

Reply:
Behavior is similar to the other global set functions in this file. We use the same for qengine as well and the expectation is that user sets this once before running the program.

Reply:
Yeah that one is also my concern. @dreiss, do you think it is ok leave as if for now?

Reply:
@kimishpatel see comment below about making this default to true on mobile. So we don't have to explicitly call this on mobile.